app/javascript/react/screens/App/Overview/components/Migrations/Migrations.js

```diff
@@ -56,6 +56,12 @@ const Migrations = ({
     setActiveFilter(eventKey);
   };
 
+  const infraNameMapper = (plans = [], mappings = []) =>
+    plans.map(plan => ({
+      ...plan,
+      infraMappingName: mappings.find(mapping => mapping.id === plan.options.config_info.transformation_mapping_id).name
+    }));
```

Comment:
Seeeee then it means pulling in the `transformationMappings` somewhere awkward (or maybe not?) so we're still only messing with it, the array, after being `asMutable`ed 🤔

Reply:
Adjusting the array in the reducer to suit our application needs has been our MO in a few other places too.
I like the fact that the data flows down from the reducer as a single source of truth and is available in the redux state when needed.

If it seems too hard of a change right now, maybe we can refactor it later to include the `infra Mapping Name` and the `state` logic (in #427) in the reducer.

Reply:
I think modifying the data in the reducer is actually higher up in the food chain 🍕 since the reducer is what supplies data to the `overview` key in the redux store.  Practically speaking, I'd say there is not much difference though.

I have definitely seen examples (in fact we do it a lot in our codebase) to manipulate data in reducers, but it is also pattern to use [redux selectors](https://redux.js.org/introduction/learning-resources#selectors) [(this page has a better code example)](https://redux.js.org/recipes/computing-derived-data) in the `mapStateToProps` call.

Now you have three choices @AllenBW